docs/glossary.rst

```diff
@@ -14,13 +14,13 @@ Glossary
    10 deg point
       The point along a topography profile at which the slope angle is 10
       degrees. Is usually considered to be the point after which the profile is
-      consistently below 10 degrees. 
+      consistently below 10 degrees.
 
    beta angle
-      Angle from the topmost point of a profile to the :term:`beta point`. 
+      Angle from the topmost point of a profile to the :term:`beta point`.
 
-   release thickness [*relTh*, *d0*]
-      TODO: which symbol 
+   release thickness [*relTh*]
+      TODO: which symbol
```

Comment:
```suggestion
   release thickness [*relTh*]
      ```

Also add the same for entrainment thickness

Reply:
done